app/helpers/organizations_helper.rb

```diff
@@ -2,4 +2,4 @@ module OrganizationsHelper
   def oganizations_index_subtitle
     t "organizations.subtitle.default" if params[:order].blank? || params[:order] == :created_at
   end
-end
+end
```

Comment:
Have you correctly configured http://editorconfig.org/ on your text editor or IDE? Please do so 🙏  https://codeclimate.com/github/AyuntamientoMadrid/agendas/app/helpers/organizations_helper.rb?from_sha=c8f2252c&to_sha=c2288a8b